test/agent/config/command_test.py

```diff
@@ -0,0 +1,775 @@
+"""
+Tests for command.py
+"""
+
+import os
+import shutil
+import pytest
+from agent.config.command import (
+    Command,
+    LLMCommand,
+    Standard,
+    Multimodal,
+    Assistant,
+    BrowserCommand,
+    Navigate,
+    BrowserFile,
+    FullPageScreenshot,
+    ElementScreenShot,
+    CollectNodes,
+    SaveHtml,
+    Sleep,
+    Click,
+    move_file,
+)
+
+# Sample test data
+standard_command_data = {"role": "user", "content": "Hello, world!"}
+
+
+def test_command_init():
+    """
+    Function to test initialization of Command
+    """
+    command = Command("llm", "test_command", {"param": "value"})
+    assert command.command_type == "llm"
+    assert command.command_name == "test_command"
+    assert command.params == {"param": "value"}
```

Comment:
I dont know if you need to test it this much. Mainly just function values, its good either way but not nessecary